content/tv/bcak/32.md

```diff
@@ -12,12 +12,13 @@ guests:
 - Kat Cosgrove
 hosts:
 - Nate Schutta
-lastmod: '2021-06-03'
+lastmod: '2021-06-21'
 minutes: 120
 title: The one with Kat Cosgrove
 truncate: ''
 twitch: vmwaretanzu
 type: tv-episode
+youtube: "luGjiUPR2f8"
 ---
 
-Please join Nate Schutta and guest Kat Cosgrove, a Developer Advocate at JFrog, and an actual cyborg. Her professional background has run the gamut from bartender, to video store clerk, to teacher, to software developer. She credits this wide-ranging experience for her success as a speaker, developer, and advocate. Her specialty is approachable 101-level content for junior developers or anyone else who considers themselves a newbie, with a particular focus on DevOps. When she's not building demos or at a conference, she spends her time playing video games, watching horror movies, and reading science fiction. She lives in Seattle with her cat, Espresso, who is the real brains behind the operation and actually ghostwriting all of her tweets." with "Before joining Redis Labs, Brian founded and ran Integrallis, a successful consultancy for over 2 decades. He is a frequent speaker at user groups and conferences nationally and abroad. Brian is the author of “Beginning POJOs: Spring, Hibernate, JBoss and Tapestry,” co-author of the “Enterprise Java Development on a Budget: Leveraging Java Open Source Technologies” and a contributor to O'Reilly's “97 Things Every Project Manager Should Know”."
\ No newline at end of file
+Please join Nate Schutta and guest Kat Cosgrove, a Developer Advocate at JFrog, and an actual cyborg. Her professional background has run the gamut from bartender, to video store clerk, to teacher, to software developer. She credits this wide-ranging experience for her success as a speaker, developer, and advocate. Her specialty is approachable 101-level content for junior developers or anyone else who considers themselves a newbie, with a particular focus on DevOps. When she's not building demos or at a conference, she spends her time playing video games, watching horror movies, and reading science fiction. She lives in Seattle with her cat, Espresso, who is the real brains behind the operation and actually ghostwriting all of her tweets." with "Before joining Redis Labs, Brian founded and ran Integrallis, a successful consultancy for over 2 decades. He is a frequent speaker at user groups and conferences nationally and abroad. Brian is the author of “Beginning POJOs: Spring, Hibernate, JBoss and Tapestry,” co-author of the “Enterprise Java Development on a Budget: Leveraging Java Open Source Technologies” and a contributor to O'Reilly's “97 Things Every Project Manager Should Know”."
```

Comment:
Looks like Brian's blurb got appended here?

Reply:
Yeah, turns out that was in the spreadsheet we maintain with the schedule...thanks for noticing!